parler/appsettings.py

```diff
@@ -17,7 +17,7 @@
 # Have to fill the default section explicitly to avoid circular imports
 PARLER_LANGUAGES.setdefault('default', {})
 PARLER_LANGUAGES['default'].setdefault('code', PARLER_DEFAULT_LANGUAGE_CODE)
-PARLER_LANGUAGES['default'].setdefault('fallback', PARLER_DEFAULT_LANGUAGE_CODE)
+PARLER_LANGUAGES['default'].setdefault('fallbacks', [PARLER_DEFAULT_LANGUAGE_CODE])
```

Comment:
I think this should take care of the fact that people may still have `'fallback': ..` in their config.